types/express/index.d.ts

```diff
@@ -15,7 +15,7 @@
 /// <reference types="serve-static" />
 
 import * as bodyParser from "body-parser";
-import * as serveStatic from "serve-static";
+import serveStatic = require("serve-static"); // FIXME See Fix express.static() with esModuleInterop https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23335
```

Comment:
I don't think the comment is needed if this fixes the issue?

Reply:
If someone passes by and finds it strange (*) to have `require` instead of `from`, he has the link to the explanation

(*) and wants to revert the change

Reply:
It's actually more correct this way -- the thing being imported is a function. See the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports).

Reply:
@andy-ms I've removed the comment